css/css-images/gradient/gradient-eval-001.tentative.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>Gradient interpolation</title>
+  <meta name="fuzzy" content="maxDifference=2">
```

Comment:
Per [the docs](https://web-platform-tests.org/writing-tests/reftests.html#fuzzy-matching), you have to provide both a maxDifference *and* a totalPixels.

(In this case, the totalPixels can just be 0-10000.)

I think you also want a maxDifference of 1-2, not just 2? At least, if I'm reading the docs correctly.